.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ commands:
               . venv/bin/activate
               # venv/ dir doesn't seem to save enough info to keep the
               # editable installation
-              pip install --progress-bar=off -e .
+              pip install --progress-bar=off -e '.[movercli]'
```

Comment:
Whoops!  This wasn't the same as the pip install below and it should have been.  This may speed up our tests a touch.